meerkat_consul/export.py

```diff
@@ -240,13 +339,32 @@ def post_events(events_payload):
     logger.debug(event_res.json().get('message'))
 
 
+@async
+def post_data_set(data_sets_payload):
+    data_set_res = post("{}/dataValueSets?importStrategy=CREATE_AND_UPDATE".format(dhis2_api_url),
+                        headers=dhis2_headers, data=json.dumps(data_sets_payload))
+    logger.info("Send batch of data entries with status: %d", data_set_res.status_code)
+    logger.debug(data_set_res.json().get('message'))
+
+
 def uuid_to_dhis2_uid(uuid):
     result = uuid[-11:]
     # DHIS2 uid needs to start with a character
     if result[0].isdigit():
         result = 'X' + result[1:]
     return result
 
+
+def get_period_from_date(input_date, formId):
```

Comment:
1) `formId' >> 'form_id'
2) This function looks too complicated.
Generally if dhis2_config['data_set_period'][form_id] doesn't exist we return None (why this happens btw?) so:
```
if dhis2_config.get('data_set_peroid', {}).has_key(form_id):
    return '{d.year}{d.month}{d.day}'.format(d=input_date) # or simpler datetime.strftime(input_date, "%d%m%Y") if you don't mind zero padded day & month
return # Why is this return None here?
```